src/components/utils/http.js

```diff
@@ -2,35 +2,37 @@ import { events } from '../../core'
 import Tracker from '../../Tracker'
 const SDK_SERVER_URL = 'https://sdk.onfido.com'
 
-const reduceObj = (object, callback, initialValue) =>
-  Object.keys(object).reduce(
-    (accumulator, key) => callback(accumulator, object[key], key, object),
-    initialValue)
-
 const handleError = ({status, response}, callback) => {
   console.error(status, response)
   Tracker.sendError(`${status} - ${response}`)
   callback()
 }
 
-export const postToServer = (payload, serverUrl, token, onSuccess, onError) => {
+export const postToServer = (payload, serverUrl, token, onSuccess, errorCallback) => {
```

Comment:
can you put this in a sdk backend specific module? together with `SDK_SERVER_URL`